setup.py

```diff
@@ -1,10 +1,10 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='wfqc',
+    name='pubmetric',
     version='0.1.0',
-    description='A description of your project',
-    author='Your Name',
+    description='Cocitation based metric for workflow evaluation',
+    author='Alma Nilsson',
```

Comment:
It's a good practice to list everyone who helped in the software development or design (not sure if anyone else apart from Magnus and myself was involved), but keep your name first of course, and add your email as `author_email`. I would suggest adding a .cff file (see https://github.com/Workflomics/workflomics-frontend/blob/main/CITATION.cff) to list the contributors and the official title of the package, so you can publish it to Zenodo. That file provides the citation metadata (unlike setup.py which is more for the package reference).